chromium_src/base/mac/foundation_util.mm

```diff
@@ -37,7 +37,7 @@
   }
 
 #if !defined(OFFICIAL_BUILD)
-  return "com.brave.Browser.development";
+  return "com.brave.Browser";
```

Comment:
Why did this need to change? Was it defaulting non-signed builds to the dev ID?

Reply:
actually this shouldn't be needed. I think it's just leftover from when I couldn't get it to work when everything was set to com.brave.Browser.development. I'll check now